src/cascadia/WindowsTerminal/main.cpp

```diff
@@ -126,7 +126,11 @@ int __stdcall wWinMain(HINSTANCE, HINSTANCE, LPWSTR, int)
     AppHost host;
     if (!host.HasWindow())
     {
-        return 0;
+        // If we were told to not have a window, exit early. Make sure to use
+        // ExitProcess to die here. If you try just `return 0`, then
+        // the XAML app host will crash during teardown. ExitProcess avoids

```

Comment:
Is the crash by design or just some sort of detect in the framework?

Reply:
I couldn't tell you. It also might be caused because we're doing weird things with the XamlHost to prevent other crashes, I really can't say.

Reply:
Fwiw, I believe the generally accepted spelling is `seppuku` in case you want to add it 😉